src/services/Store/store/persist.config.ts

```diff
@@ -168,14 +169,31 @@ export const migrations = {
       ...state,
       settings: dissoc('inactivityTimer', state.settings)
     };
+  },
+  6: (state: DataStore) => {
+    return {
+      ...state,
+      // @ts-expect-error dPath is present on data to be migrated, want to remove it
+      accounts: state.accounts.map(({ dPath, ...account }) => {
+        if (!dPath) {
+          return account;
+        }
+        const [path, index] = getDerivationPath(dPath)!;
+        return {
+          ...account,
+          path,
+          index
+        };
```

Comment:
This can break if `dPath` is a custom derivation path. The `getDerivationPath` function only works for the derivation paths defined in the library.